studio/pages/project/[ref]/auth/settings-new-all.tsx

```diff
@@ -0,0 +1,1109 @@
+import {
```

Comment:
can we break down this page into smaller components - 1,000 line pages will take long to re-render for local dev

Reply:
this is unused file. needs deleting